### PR TITLE
6280: Add compare page

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/homepage/ui.test.js [hunk 1]
```diff
@@ -226,6 +252,19 @@ test('should render the compare promo', async (t) => {
     .expect(await extractInnerText(promo.find('p'))).eql(content.comparePromoDescription);
 });
 
+test('should navigate to the compare page when the compare promo is clicked', async (t) => {
+  await pageSetup({ t, cookiePayload: { id: '88421113', name: 'Cool Dude', organisation: 'view' } });
+
+  const adminPromo = Selector('[data-test-id="compare-promo"] a');
```
Comment (alibombaye) [inline app/pages/homepage/ui.test.js:258]: rename to `comparePromo`

File: app/pages/homepage/ui.test.js [hunk 2]
```diff
@@ -226,6 +252,19 @@ test('should render the compare promo', async (t) => {
     .expect(await extractInnerText(promo.find('p'))).eql(content.comparePromoDescription);
 });
 
+test('should navigate to the compare page when the compare promo is clicked', async (t) => {
+  await pageSetup({ t, cookiePayload: { id: '88421113', name: 'Cool Dude', organisation: 'view' } });
```
Comment (alibombaye) [inline app/pages/homepage/ui.test.js:256]: can we remove the cookiePayload. Not really needed for this. Looks bit misleading

Comment (kat1906) [reply app/pages/homepage/ui.test.js:256]: oh yeah I removed it from the others

File: app/pages/homepage/ui.test.js (hunk 1 above)
Comment (kat1906) [reply app/pages/homepage/ui.test.js:258]: 👀

**Pull request merged**